lighthouse-core/audits/redirects-http.js

```diff
@@ -37,13 +37,6 @@ class RedirectsHTTP extends Audit {
    * @return {!AuditResult}
    */
   static audit(artifacts) {
-    if (!artifacts.HTTPRedirect ||
-        !artifacts.HTTPRedirect.value) {
```

Comment:
I was trying to only do shape checks of the artifact itself, not the value/debugString/whatever stuff used for artifact classification.

In this case it was just a drive by fix because `if (!artifacts.HTTPRedirect.value)` evaluated as true, you'd get the same exact output as just letting it go through the default path below